pandas/core/frame.py

```diff
@@ -4367,6 +4367,10 @@ def unstack(self, level=-1, fill_value=None):
         Name to use for the 'value' column.
     col_level : int or string, optional
         If columns are a MultiIndex then use this level to melt.
+    keep_index : boolean, optional, default False
```

Comment:
this is commonly called ``index=False`` everywhere else.

add a versionadded

Reply:
So better to just name it ```index```  and if True resulting in the original index with duplicate entries? What about the option @TomAugspurger proposed?